docker-argocd-app-diff/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM argoproj/argocd:v1.7.7
```

Comment:
Why did you rollback from 1.8.1 to 1.7.7?

Reply:
fix with 95c6999